chopsticks-sample/src/main/resources/dev/chopsticks/sample/app/dstreams/dstreams-sample-worker-app.conf

```diff
@@ -0,0 +1,23 @@
+akka {
+  coordinated-shutdown.run-by-jvm-shutdown-hook = on
```

Comment:
`include required(classpath("app-shared.conf"))` instead, which already sets this?